acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/proxy/PassThroughAcceptanceTest.java

```diff
@@ -33,24 +33,33 @@
 
   @Test
   public void ethGetTransactionCountReturnCorrectNumber() {
-    final BigInteger startTransactionCount =
+    final BigInteger pantheonTransactionCount =
         ethNode().transactions().count(richBenefactor().address());
-    submitTransactionAndWaitForBlock();
+    BigInteger ethSignerTransactionCount =
+        ethSigner().transactions().count(richBenefactor().address());
 
-    final BigInteger endTransactionCount =
-        ethNode().transactions().count(richBenefactor().address());
+    assertThat(pantheonTransactionCount).isEqualTo(ethSignerTransactionCount);
+
+    submitTransactionAndWaitForBlock(); // should increase the transaction count by one.
+
+    ethSignerTransactionCount = ethSigner().transactions().count(richBenefactor().address());
 
-    assertThat(startTransactionCount.add(BigInteger.ONE)).isEqualTo(endTransactionCount);
+    assertThat(pantheonTransactionCount.add(BigInteger.ONE)).isEqualTo(ethSignerTransactionCount);
   }
 
   @Test
-  public void ethBalanceRequesReturnsCorrectBalance() {
-    final BigInteger startBalance = ethNode().accounts().balance(RECIPIENT);
+  public void ethBalanceRequestReturnsCorrectBalance() {
+    final BigInteger startBalance = ethSigner().accounts().balance(RECIPIENT);
```

Comment:
should this be ethSignerStartBalance (for symmetry with line 56).

Reply:
done